### PR TITLE
feat(twicpics): fix modifiers and improve documentation

### PR DESCRIPTION
**Pull request by mbgspcii**
TwicPics provider: 
- fixes `fit=contain` feature
- fixes `fit=inside` feature
- fixes `fit=outside` feature
- additional documentation and playground samples

### CODE REVIEW

File: docs/content/en/4.providers/twicpics.md [hunk 1]
```diff
@@ -5,16 +5,437 @@ navigation:
...
+```
+
+This will return a variant of your image for which we have, in order:
+1. placed the focus on the center of interest of the original image
+2. then cropped the image to 16:9 from the center of interest
+3. then rotated the image 90° to the left
+
+The result is a cropped image with the area of interest retained and displayed in 16:9 format.
+```html
+<nuxt-img
```
Comment (Atinux) [inline docs/content/en/4.providers/twicpics.md:396]:
```suggestion
<NuxtImg
```

File: docs/content/en/4.providers/twicpics.md [hunk 2]
```diff
@@ -5,16 +5,437 @@ navigation:
...
+Example:
+
+This will return a variant of image for which we have, in order:
+1. cropped the image from the center to 16:9 aspect ratio
+2. then placed the focus on the center of interest of the cropped image
+3. then rotate the image 90° to the left
+
+The result is a 9:16 (not 16:9) image with a possibly false area of interest.
+```html
+<nuxt-img
```
Comment (Atinux) [inline docs/content/en/4.providers/twicpics.md:382]:
```suggestion
<NuxtImg
```

File: docs/content/en/4.providers/twicpics.md [hunk 3]
```diff
@@ -5,16 +5,437 @@ navigation:
...
+### zoom
+
+Zooms into the image by a factor equal or superior to 1 towards the focus point while preserving the image size.
+
+Syntax: `{ zoom: `[`number`](https://www.twicpics.com/docs/api/manipulations/?utm_source=nuxt&utm_medium=organic&utm_campaign=provider#number)` }`
+
+```html
+<NuxtImg
+  provider="twicpics"
+  src="/components/flip/cherry-3.jpg"
```
Comment (Atinux) [inline docs/content/en/4.providers/twicpics.md:341]:
```suggestion
  src="/cherry-3.jpg"
```

Do you mind also removing the `/components/flip` part on other examples?

It is not necessary.

File: docs/content/en/4.providers/twicpics.md [hunk 4]
```diff
@@ -5,16 +5,437 @@ navigation:
...
+
+Using the TwicPics Provider you can, out of the box, benefit from at least :
+- performance of our network: global CDN, optimized protocols and competitive caching
+- ideal compression: modern technology and Next-Gen formats (TwicPics delivers [`WebP`](https://en.wikipedia.org/wiki/WebP) natively for accounting browsers and can also delivers [`avif`](https://en.wikipedia.org/wiki/AVIF))
+
+And using the TwicPics API, you will be able to access all these features: [smart cropping with TwicPics focus auto](#focus), [true color](#truecolor), [flip](#flip), [turn](#turn), [crop](#crop), [zoom](#zoom).
+
+TwicPics also offers additional functionalities through its own [Nuxt components](#go-further-with-twicpics): [LQIP](https://www.twicpics.com/blog/blurred-placeholders#how-to-use-the-blurred-preview-feature), [CLS optimization](https://www.twicpics.com/blog/images-and-core-web-vitals#images-and-cumulative-layout-shift), [content driven and lazy-loading](https://www.twicpics.com/features), and video handling.
```
Comment (Atinux) [inline docs/content/en/4.providers/twicpics.md:20]: I would remove this one since it is a bit too much marketing + SEO linking 👀

**Review by Atinux — COMMENTED** [review]
Added some comments

File: docs/content/en/4.providers/twicpics.md (hunk 1 above)
Comment (mbgspcii) [reply docs/content/en/4.providers/twicpics.md:394]: > Added some comments, thank you for this PR ❤️

Thanks for this early review. :thumbsup:

**Review by pi0 — APPROVED** [review]

**Comment by pi0** [comment]
Thanks @mbgspcii @Atinux for working on this 💯

**Pull request merged**